### PR TITLE
Failover database and elastic pools Net SDK

### PR DESCRIPTION
**Pull request by emgu-ms**
REST API commit change - https://github.com/Azure/azure-rest-api-specs/commit/597f7dd325c864dd4db48d9374451c16e0337158#diff-cf76f9fb3a55909fa9cd42d1e4612784

Added database and elastic pool failover to Net SDK.  Also included tests.

### CODE REVIEW

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj [hunk 1]
```diff
@@ -13,6 +13,7 @@
 			<![CDATA[
 New features:
 - Added support for serverless databases.
```
Comment (jaredmoo) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj]: Line 15 is release notes for 1.31.0-preview, this was already published (see https://www.nuget.org/packages/Microsoft.Azure.Management.Sql). So you should update version to 1.32 (here and in AssemblyInfo.cs) and wipe out the old release notes.

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/tests/FailoverTests.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                string dbName = SqlManagementTestUtilities.GenerateName();
+                var dbInput = new Database()
+                {
+                    Location = server.Location
+                };
+                var db = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, dbName, dbInput);
+                Assert.NotNull(db);
+
+                // Failover database
+                AzureOperationResponse failoverResponse = await sqlClient.Databases.BeginFailoverWithHttpMessagesAsync(
```
Comment (jaredmoo) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/tests/FailoverTests.cs]: just call `Failover` instead, that way you are verifying the entire e2e operation succeeds.

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/tests/FailoverTests.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                var dbInput = new Database()
+                {
+                    Location = server.Location,
+                    ElasticPoolId = ep.Id
+                };
+                var db = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, dbName, dbInput);
+                Assert.NotNull(db);
+
+                // Failover elastic pool
+                AzureOperationResponse failoverResponse = await sqlClient.ElasticPools.BeginFailoverWithHttpMessagesAsync(
```
Comment (jaredmoo) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/tests/FailoverTests.cs]: also use `Failover` here.

**Review by jaredmoo — CHANGES_REQUESTED** [review]

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj [hunk 4]
```diff
@@ -13,6 +13,7 @@
...
+- Added support for failover databases and elastic pools.
       ]]>
 		</PackageReleaseNotes>
 	</PropertyGroup>
```
Comment (erich-wang) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj]: @emgu-ms, you need to update Version if you want to publish new nuget pcakge

Comment (erich-wang) [reply sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj]: You may also need update the value of AssemblyFileVersion.

**Comment by erich-wang** [comment]
Please resolve conflicts as well.

**Review by erich-wang — CHANGES_REQUESTED** [review]
Please resolve conflicts.

File: sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj [hunk 5]
```diff
@@ -12,6 +12,7 @@
 		<PackageReleaseNotes>
 			<![CDATA[
 New features:
+- Added support for failover databases and elastic pools.
```
Comment (jaredmoo) [inline sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj]: Sorry, looks like 1.32 was just published. Can you wipe out the 1.32 release notes and bump to 1.33?

**Review by jaredmoo — APPROVED** [review]

File: eng/mgmt/mgmtmetadata/sql_resource-manager.txt [hunk 6]
```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
...
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=.
-2019-07-11 15:27:39 UTC
+cmd.exe /c autorest.cmd https://github.com/emgu-ms/azure-rest-api-specs/blob/failoverSwaggerChanges/specification/sql/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\GitCode\azure-sdk-for-net\sdk
+2019-07-12 23:19:41 UTC
 Azure-rest-api-specs repository information
-GitHub fork: Azure
-Branch:      master
-Commit:      808cfe2da58aa6f1d93bed748004dfb97268e6e3
+GitHub fork: emgu-ms
```
Comment (jaredmoo) [inline eng/mgmt/mgmtmetadata/sql_resource-manager.txt]: Aren't your changes checked into master? Published SDK must be based on azure/master.

**Review by jaredmoo — COMMENTED** [review]
.

**Review by jaredmoo — APPROVED** [review]

**Review by erich-wang — APPROVED** [review]

**Pull request merged**